appinventor/appengine/src/com/google/appinventor/client/OdeMessages.java

```diff
@@ -655,15 +655,15 @@ public interface OdeMessages extends Messages, AutogeneratedOdeMessages {
   @Description("Message providing details about starting a USB connection.")
   String usbMenuItem();
 
-  @DefaultMessage("Reset Connection")
+  @DefaultMessage("Reset Connection (Alt + Shift + R)")
```

Comment:
We may want to revisit this choice, since Alt on Windows typically maps to Cmd on macOS, and Cmd + R will refresh the page.

Reply:
> We may want to revisit this choice, since Alt on Windows typically maps to Cmd on macOS, and Cmd + R will refresh the page.

Does't Ctrl usually map to Cmd on MacOS in GUI aps? I usually expect Alt to map to Option.

I realize that we have a Ctrl button, and that works as Ctrl for a lot of command line stuff. There's no consistent official mapping.

Reply:
Thanks for the feedback @ewpatton. I will look into these and fix them.
I will once go through the shortcuts in macOS and see for any more overlaps.
I still need to add code that adjusts the shortcut key messages based on whether the user is on Windows or macOS.
The community feedback is still pending, and I plan to complete it by the end of this week.